src/interp/interp.h

```diff
@@ -503,8 +502,12 @@ class Store {
  private:
   template <typename T>
   friend class RefPtr;
+  friend class Thread;
```

Comment:
I did this for simplicity, but I am open to other suggestions.

Reply:
It this still needed?